### DISCUSSION TITLE
H265 RTSP stream to .mp4 without transcoding

### DISCUSSION THREAD

**Discussion by yngvesl** [bytedeco/javacv#2166] [Q&A]
Is there a way to use the copy codec with the FFmpegFrameRecorder on a H.265 RTSP stream and feed the stream into a .mp4 container?

**Comment by saudet** [comment]
Probably with a few modifications, yes

**Comment by saudet** [comment]
Duplicate of https://github.com/bytedeco/javacv/issues/1953
